src/module/storage/NetworkStorage.ts

```diff
@@ -124,7 +124,17 @@ export const NetworkStorage = {
         const networks = NetworkStorage.getStorage();
         const masterUuid = Helpers.uuidForStorage(master.uuid);
         const slaveUuids = networks[masterUuid] ?? [];
-        return slaveUuids.map(uuid => fromUuidSync(Helpers.uuidFromStorage(uuid))) as (SR5Actor | SR5Item)[];
+        const slaves: (SR5Actor | SR5Item)[] = [];
+        for (const uuid of slaveUuids) {
+            const item = fromUuidSync(Helpers.uuidFromStorage(uuid));
+            // in case we have a stale id in storage, only add valid items
+            if (item) {
```

Comment:
@taMiF this check here ensures we don't get `null` values getting sent to the actor sheet. With this fix in I was able to still open character sheets when there was null data, then I went about fixing the actual problem.

If you wanted to reproduce, (without using this branch) add an unlinked actor to a scene as a token. Then connect them to a Grid, then delete them. That will make other actors connected to that grid from openning. 
The GRID item itself could be opened and the "Clear All" functionality would work, thus removing the stale entry.